src/styles.module.css

```diff
@@ -0,0 +1,141 @@
+:root {
```

Comment:
please, use the scope of tool element instead of document to prevent conflicts.

For example:

```css
.nft-tool {
 --border: 1px solid var(--color-border);
}
```